### PR TITLE
Add Authorization rules

### PR DESCRIPTION
**Pull request by tufoinnkuo10**
In this project, the following was done;
-  Install CanCanCan in your project.
-  Add a role column to the users table. Remember to use a migration for this.
          *A user can delete a post if it is theirs or if they have an admin role (column role has value "admin"). Use CanCanCan for this authorization.
         *For that you need to implement the post deleting functionality. Add the "Delete" button to the view and make sure that only authorized users can see it.
        *A user can delete a comment if it is theirs or if they have an admin role (column role has value "admin"). Use CanCanCan for this authorization.
        *For that you need to implement the comment deleting functionality. Add the "Delete" button to the view and make sure that only authorized users can see it.

### CODE REVIEW

File: db/migrate/20220203135812_create_users.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    create_table :users do |t|
+      t.text :name, null: false
+      t.string :photo
+      t.string :bio
+      t.string :role
+      t.integer :posts_counter, default: 0
+      t.timestamps
+    end
+  end
+end
```
Comment (DeliceLydia) [inline db/migrate/20220203135812_create_users.rb]: - [ ] There is an error in your migrations some have the same name, please fix that
e

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @tufoinnkuo10, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Major highlights for your project:
- Good README
- Followed git workflow
- Descriptive PR


## Required Changes ♻️
_Check the comments under the review._




### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Approved🎉🎉 

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HENRYKC24** [comment]
Thank you very much @DeliceLydia :pray: We will now go ahead to merge as you directed:running:

**Pull request merged**